tools/topology/topology2/platform/intel/sdw-jack-generic.conf

```diff
@@ -186,56 +186,58 @@ Object.Pipeline.host-gateway-capture [
 		Object.Widget.host-copier.1 {
 			stream_name	"Passthrough Capture 0"
 			pcm_id 1
-			num_input_audio_formats 9
-			num_output_audio_formats 9
```

Comment:
and that's also clearly wrong @ranj063. I don't know how we identified 9 formats below.

Reply:
> and that's also clearly wrong @ranj063. I don't know how we identified 9 formats below.

The first 6 supported formats are already defined in [host-gateway-capture.conf](https://github.com/thesofproject/sof/blob/main/tools/topology/topology2/include/pipelines/cavs/host-gateway-capture.conf). So, we just define the 7th to 9th supported formats here.

Reply:
@plbossart that was a mistake. We cannot maintain this for sure. We should have defined all 9 formats here. Anyway, the reason why the audio_format class has been left into these files is because I wanted to use the new DefineArrays feature for these because we have quite a few. Do you mind if I take over this PR?

Reply:
> Do you mind if I take over this PR?

Be my guest and have at it! I just started this out of frustration when trying to add the use_chain_dma stuff, I am all for doing the 'right thing'(tm) here.